portfolio/src/main/webapp/script.js

```diff
@@ -102,3 +102,18 @@ $(document).ready(() => {
     fadeDiv('.project');
   });
 });
+
+/**
+ * Adds a greeting to the page.
+ */
+function addGreeting() {
+  fetch('/data').then(response => response.text()).then((greeting) => {
+    // Convert the HTML string into a document object
+	  let parser = new DOMParser();
+	  let greetingObject = parser.parseFromString(greeting, 'text/html');
+
+    // Add H1 greeting object to HTML page
+    let greetingContainer = document.getElementById('greeting-container');
+    greetingContainer.append(greetingObject.documentElement);
```

Comment:
this will append repeated greetings if a user clicks the button multiple times. Is that intended? Either way is fine, just want to make sure it's working the way you expect.

Reply:
Yes, that's expected behavior -- thanks for double-checking that.